content/en/docs/concepts/security/security-checklist.md

```diff
@@ -0,0 +1,333 @@
+  enabled.
+- [ ] The root certificate is protected (either an offline CA, or a managed
+  online CA with effective access controls).
+- [ ] Intermediate and leaf certificates have an expiry date no more than 3
+  years in the future.
+- [ ] A process exists for periodic access review, and reviews occur no more
+  than 24 months apart.
+
+See the [Role Based Access Control Good Practices](/docs/concepts/security/rbac-good-practices/)
+for guidance related to authentication and authorization.
+
+After bootstrapping, neither users nor components should authenticate to the
+Kubernetes API as `system:masters`. Similarly, running all of
+kube-controller-manager as `system:masters` should be avoided. In fact,
+`system:masters` should only be used as a break-glass mechanism, as opposed to
+an admin user.
+
+## Network security
+
+- [ ] CNI plugins in-use supports network policies.
+- [ ] Network policies to filter how workloads and services can communicate
+  inside of the cluster are enabled.
+- [ ] Use of a service mesh to encrypt all communications inside of the cluster.
+- [ ] The Kubernetes API is not exposed publicly on Internet.
+- [ ] Access from the workloads to the cloud metadata API is filtered.
+
+A number of [Container Network Interface (CNI) plugins](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/)
+plugins provide the functionality to
+restrict network resources pods may communicate with. This is most commonly done
+through [Network Policies](/docs/concepts/services-networking/network-policies/)
+which provide a namespaced resource to define rules.
+
+Not all CNI plugins provide encryption in transit. If the chosen plugin lacks this
+feature, an alternative solution could be to use a service mesh to provide that
+functionality.
+
+The `etcd` datastore of the control plane should have controls to limit access.
+Furthermore, mutual TLS (mTLS) should be used to communicate securely with it.
+The certificate authority for this should be unique to `etcd`.
+
+External Internet access to the Kubernetes API server should be restricted
+to not expose the API publicly. You can then use a bastion host to access the server.
+
+If a cloud provider is used for hosting Kubernetes, the access from pods to the cloud
+metadata API `169.254.169.254` should also be restricted or blocked if not needed
+because it may leak information.
+
+## Pod security
+
+- [ ] RBAC right to `create` workloads is only granted if necessary.
+- [ ] Appropriate pod security is enforced for all workloads.
+- [ ] Memory and CPU limits are set for the workloads.
+- [ ] For nodes that support it, Seccomp is enabled with appropriate syscalls
+  profile for applications written or generated.
+- [ ] For nodes that support it, AppArmor is enabled with appropriate profile
+  for applications written or generated.
+
+
+RBAC authorization is crucial but
+[cannot be specific enough on the Pods resource](/docs/concepts/security/rbac-good-practices/#workload-creation)
```

Comment:
Yes, this is detailed in the following sentences and in the hyperlinks. I'm not sure how I could rephrase it.

Reply:
I think we can just rephrase it just the way I said it, tbh (unless you have alternate suggestions)

```suggestion
[cannot be specific to pods, resources, anything that manages pods](/docs/concepts/security/rbac-good-practices/#workload-creation)
```

Reply:
I'm not sure what you imply by putting "resources" apart like that. The idea was to explain that RBAC cannot be granular enough to have authorization on the pod security part in the PodSpec. And because it concerns pods, it also concerns all the resources that will end in the creation of a pod, like a Deployment, a DaemonSet, etc...

Reply:
I am sorry that was my bad. Does this make sense @mtardy ?
```suggestion
[cannot be granular enough to have authorization on the Pods' resources](/docs/concepts/security/rbac-good-practices/#workload-creation)
```

Reply:
Thanks a lot!